app/containers/MessageBox/index.js

```diff
@@ -189,7 +189,7 @@ class MessageBox extends Component {
 		this.setInput(text);
 
 		if (this.component) {
-			const { start, end } = this.component._lastNativeSelection;
+			const { start, end } = this.component._lastNativeSelection || { start: 0, end: 0 };
```

Comment:
Doesn't it looks like a workaround instead of a fix?
I mean, when this code run `this.component. _lastNativeSelection` is empty, right?
If it's empty, isn't it better if we test `text`?

Just change from:
```jsx
if (this.component) {
```

to:
```jsx
if (!isTextEmpty) {
```

What do you think?

Reply:
Nice!!